.github/workflows/main.yml

```diff
@@ -1,21 +1,22 @@
 # This is a basic workflow to help you get started with Actions
 
 name: CI
-
+ 
 # Controls when the action will run.
 on:
   # Triggers the workflow on push or pull request events but only for the master branch
   push:
     branches:
       - master
+      - patch-1
```

Comment:
Could you explain this change?

Reply:
That's just for testing. Before I merge, it'll be reverted.